diff --git a/ocp.toml b/ocp.toml
index ce5a3853..af4826ed 100644
--- a/ocp.toml
+++ b/ocp.toml
@@ -393,16 +393,6 @@ class Adaptor3d_Surface;
 [OSX]
     modules = ["Cocoa"]
     symbols = "symbols_mangled_mac.dat"
-    prefix = "/usr/local/miniconda/envs/cpp-py-bindgen"
-    includes = ['/usr/local/Cellar/gcc@8/8.3.0_2/lib/gcc/8/gcc/x86_64-apple-darwin19/8.3.0/../../../../../../include/c++/8.3.0',
-                '/usr/local/Cellar/gcc@8/8.3.0_2/lib/gcc/8/gcc/x86_64-apple-darwin19/8.3.0/../../../../../../include/c++/8.3.0/x86_64-apple-darwin19',
-                '/usr/local/Cellar/gcc@8/8.3.0_2/lib/gcc/8/gcc/x86_64-apple-darwin19/8.3.0/../../../../../../include/c++/8.3.0/backward',
-                '/usr/local/Cellar/gcc@8/8.3.0_2/lib/gcc/8/gcc/x86_64-apple-darwin19/8.3.0/include',
-                '/usr/local/Cellar/gcc@8/8.3.0_2/lib/gcc/8/gcc/x86_64-apple-darwin19/8.3.0/include-fixed',
-                '/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include',
-                '/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks',
-                '/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks//OpenGL.framework/Versions/A/Headers/',
-                '/usr/local/include']
 
 [Operators]
 
